apps/backend/api/services/Search/util.js

```diff
@@ -75,21 +76,28 @@ export const filterAndSortPosts = curry((opts, q) => {
     })
   }
 
-  if (afterTime && beforeTime) {
+  if (calendarViewMode) {
```

Comment:
hmmm this isn't what I was suggesting. I was suggesting we dont need a special condition for calendars, you can just have the if (afterTime) become a separate condition from if (beforeTime), then we should be able to handle the calendar queries without any special conditions. no?

Reply:
also, does this work? if you add a new query param on the backend then it would need to be added to the schema.graphql and probably in several other places too. I think we should be fine with just beforeTime and afterTime